### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by bstrauss84**
Added node cordone/drain tasks to iptables sample.

### CODE REVIEW

**Comment by andykrohg** [comment]
lgtm! Before merging this, can we update the README to indicate:
* how to create a dedicated one-node machineset for this with a taint, and
* an updated `oc run` command to ensure the privileged pod gets scheduled on our tainted node

**Comment by andykrohg** [comment]
Another thought - maybe we could build the node drain into a secondary step in an Ansible Workflow Template that requires approval? So order could be
1. Kill the pod, and remove user rolebindings/clusterrolebindings
2. Prompt in ansible workflow for manual approval to continue
3. If yes, drain the node. (This would require a separate playbook)

Thoughts @bstrauss84 @roller1187 ?

File: samples/iptables-privileged-pod/main.yml [hunk 1]
```diff
@@ -59,3 +59,21 @@
...
+      namespace: "{{ pod_namespace }}"
+      name: "{{ pod_name }}"
+    register: pod_info
+
+  - name: Remove existing node_name file if it exists
+    file:
+      path: "/tmp/node_name.txt"
+      state: absent
+
+# TODO: VERIFY. I know for our demo we are only spinning up one pod, but I believe the k8s module iss returning a list, so I'm specifying the first result in the list just to be safe, but need to test this. #
```
Comment (andykrohg) [inline samples/iptables-privileged-pod/main.yml]: Does this task make sense to have here? The temp file will get blown away due to the nature of Automation Controller running the jobs in ephemeral EEs

Comment (bstrauss84) [reply samples/iptables-privileged-pod/main.yml]: I figured that was the case, but was hoping it still held onto it if part of a workflow.  I haven't had much luck with other methods of passing the parameter betwixt two plays.

Comment (bstrauss84) [reply samples/iptables-privileged-pod/main.yml]: Actually... I kinda wanna try using "set_stats" (https://docs.ansible.com/automation-controller/latest/html/userguide/workflows.html).  Standby...

**Review by andykrohg — APPROVED** [review]

**Comment by andykrohg** [comment]
lgtm

**Pull request merged**